moped-editor/src/views/projects/newProjectView/DefineProjectForm.js

```diff
@@ -81,6 +92,31 @@ const DefineProjectForm = ({
           />
         </Grid>
       </Grid>
+
+      <Grid container spacing={3} style={{ margin: 20 }}>
+        <Grid item xs={3}>
+          <Autocomplete
```

Comment:
<img width="436" alt="image" src="https://user-images.githubusercontent.com/3653206/171945124-c0cf16b1-1583-427e-af7a-eb26f7251aa4.png">

This is so tiny, so ignore me, but can these be in the following order? 

Signal New, Signal Mod, PHB New, PHB Mod? 

I guess just so the two align, new then mod or mod then new.

Reply:
+1 it would be nice to see these sorted alphabetically